grails-app/taglib/org/pih/warehouse/ScriptTagLib.groovy

```diff
@@ -0,0 +1,13 @@
+package org.pih.warehouse
+
+class ScriptTagLib {
+
+    Closure jquery = { Map attrs, Closure body ->
+        StringBuilder script = new StringBuilder()
+        script.append("<script src=\"https://code.jquery.com/jquery-3.3.1.slim.min.js\"")
+        script.append(" integrity=\"sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo\"")
+        script.append(" crossorigin=\"anonymous\">")
+        script.append("</script>")
```

Comment:
I decided to use `StringBuilder` for a better visibility, instead of either using `+` or to paste whole script in one line